testing/e2e/app/routes/DropdownMenu/route.tsx

```diff
@@ -2,23 +2,33 @@ import {
   Button,
   Checkbox,
   DropdownMenu,
+  IconButton,
+  List,
+  ListItem,
   MenuDivider,
   MenuExtraContent,
   MenuItem,
+  Surface,
 } from '@itwin/itwinui-react';
 import { useSearchParams } from '@remix-run/react';
+import { SvgMore } from 'node_modules/@itwin/itwinui-react/esm/utils/index.js';
```

Comment:
Do not rely on these internal paths; instead import from `@itwin/itwinui-icons-react` if you need this icon.